src/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -138,6 +139,7 @@ const Conversation: React.FC<ConversationProps> = props => {
         </Sans>
         <InfoCircleIcon />
       </StyledHeader>
+      <Spacer mt={"45px"} />
```

Comment:
Not ideal really, but this is to push the messages down below the inbox header (not pictured in the PR)

Reply:
Lil nit, no need for `{}`

Reply:
Ha, good catch. 

```suggestion
      <Spacer mt="45px" />
```